IL1-SCIM.md

```diff
@@ -0,0 +1,149 @@
+# Abstract
+The IPSIE IL1 SCIM 2.0 Profile defines a profile of SCIM 2.0 intended to meet the security and interoperability requirements of enterprise integrations using SCIM. It establishes a clear baseline for provisioning and lifecycle management of enterprise users and the roles those users are assigned. The IL1 specification ensures that CRUD operations initiate from the Identity Service and that local modifications within the Application do not occur.
+
+
+# 1.0 Introduction
+This document defines the IPSIE Identity Lifecycle 1 (SL1) Profile for SCIM 2.0. It provides a clear reference for SCIM deployments that require a well-defined security baseline meeting best practices for interoperable enterprise identity management.
+
+The profile addresses critical aspects of secure identity management, with particular emphasis on:
+* Client Authentication
+* Creation, Update, and Deletion of accounts
+* Synchronization of data from the Identity Service to the Application
+
+By adhering to this profile, organizations can implement SCIM-based integrations that meet stringent security requirements while ensuring interoperability across different implementations.
+
+# 2.0 Conventions and Definitions
+The keywords "shall", "shall not", "should", "should not", "may", and "can" in this document are to be interpreted as described in ISO Directive Part 2 [ISODIR2]. These keywords are not used as dictionary terms such that any occurrence of them shall be interpreted as keywords and are not to be interpreted with their natural language meanings.
+
```

Comment:
We'll want to normalize these to IETF style language using uppercase terminology in a future version.